src/axom/quest/DistributedClosestPoint.hpp

```diff
@@ -442,8 +442,12 @@ class DistributedClosestPointImpl
       conduit::relay::io::save(n, fname, protocol);
     };
 
+    const bool has_query_points = mesh_node.has_child("coordsets");
+
     // create conduit node containing data that has to xfer between ranks
+    // xfer_node will be empty if there are no query points on this rank
```

Comment:
What if "coordsets" is present but has size zero?  Will it be a problem that `coords` is empty?  I think yes, but need to confirm.